tests/unit_tests/dft/source/descriptor_tests.cpp

```diff
@@ -152,15 +150,15 @@ inline void set_and_get_strides(sycl::queue& sycl_queue) {
 }
 
 template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain>
-inline void set_and_get_values(sycl::queue& sycl_queue) {
+static void set_and_get_values() {
     oneapi::mkl::dft::descriptor<precision, domain> descriptor{ default_1d_lengths };
 
     using Precision_Type =
         typename std::conditional_t<precision == oneapi::mkl::dft::precision::SINGLE, float,
                                     double>;
 
     {
-        Precision_Type forward_scale_set_value{ 143.5 };
+        auto forward_scale_set_value = Precision_Type(143.5);
```

Comment:
I'm fine with this pattern, but can I ask why you changed it?

Reply:
This is giving a warning `implicit conversion between floating point types of different sizes [-Wimplicit-float-size-conversion]` otherwise.

Reply:
Interesting. thank you.